lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -791,13 +791,6 @@ static FuncDecl *deriveEncodable_encode(TypeChecker &tc, Decl *parentDecl,
   encodeDecl->setInterfaceType(interfaceType);
   encodeDecl->setAccess(std::max(target->getFormalAccess(),
                                  AccessLevel::Internal));
-
-  // If the type was not imported, the derived conformance is either from the
-  // type itself or an extension, in which case we will emit the declaration
-  // normally.
-  if (target->hasClangNode())
-    tc.Context.addExternalDecl(encodeDecl);
```

Comment:
We're never going to synthesize conformance for imported types; might as well remove this.